lib/rspec/matchers/built_in/contain_exactly.rb

```diff
@@ -154,7 +154,7 @@ def unmatched_item_count
             def +(derived_candidate_solution)
               self.class.new(
                 unmatched_expected_indexes + derived_candidate_solution.unmatched_expected_indexes,
-                unmatched_actual_indexes   + derived_candidate_solution.unmatched_actual_indexes,
+                unmatched_actual_indexes + derived_candidate_solution.unmatched_actual_indexes,
```

Comment:
I'm all for disabling that cop. Aesthetic whitespace is not a good reason for failing builds IMO - if it's really bad-looking or distractingly inconsistent I think human reviewers are better judges than rubocop.